failure looks real:

```
======================================================================
FAIL: custom.icds_reports.ucr.tests.test_expressions:TestGetAppVersion.test_cases
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/icds_reports/ucr/tests/test_expressions.py", line 195, in test_cases
    "app_version_string": "CommCare Android, version 2.36.2(433756). App v105"}))
AssertionError: 105 != 10
----------------------------------------------------------------------
```

--- a/custom/icds_reports/ucr/tests/test_expressions.py
+++ b/custom/icds_reports/ucr/tests/test_expressions.py
@@ -192,4 +192,4 @@ class TestGetAppVersion(SimpleTestCase):
             "app_version_string": "CommCare Android, version 2.36.2(433756). App v9969. "
                                   "CommCare Version 2.36. Build 433756, built on: 2017-06-23"}))
         self.assertEqual(105, expression({
-            "app_version_string": "CommCare Android, version 2.36.2(433756). App v105"}))
+            "app_version_string": "CommCare Android, version 2.36.2(433756). App v105."}))

--- a/custom/icds_reports/ucr/tests/test_expressions.py
+++ b/custom/icds_reports/ucr/tests/test_expressions.py
@@ -191,5 +191,3 @@ class TestGetAppVersion(SimpleTestCase):
         self.assertEqual(9969, expression({
             "app_version_string": "CommCare Android, version 2.36.2(433756). App v9969. "
                                   "CommCare Version 2.36. Build 433756, built on: 2017-06-23"}))
-        self.assertEqual(105, expression({
-            "app_version_string": "CommCare Android, version 2.36.2(433756). App v105."}))

